src/TraceEvent/Properties/AssemblyInfo.cs

```diff
@@ -17,3 +17,6 @@
 [assembly: NeutralResourcesLanguage("en")]
 
 // [assembly: InternalsVisibleTo("PerfView")]
+[assembly: AssemblyVersion("1.0.42.0")]
```

Comment:
Agreed. Which is exactly why a couple groups of people developed a NuGet package to establish a simple pattern you can follow and then basically forget about versioning except when you really want to bump the major (or minor) versions.
Check out [Nerdbank.GitVersioning](https://github.com/aarnott/nerdbank.gitversioning). Disclosure: I wrote it.